### PR TITLE
[ADD] Add new module mrp_repair_payment_term

### PR DESCRIPTION
**Pull request by hurrinico**
With this module we add a new field payment term on repair order.
I'm still working on porting modules on the new API

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please rename to singular

**Comment by hurrinico** [comment]
@pedrobaeza i have made a mistake with branch name but module name is mrp_repair_payment_term

**Comment by pedrobaeza** [comment]
Cool, thanks. Please check Travis status.

File: mrp_repair_payment_term/static/description/index.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<section class="oe_container">
```
Comment (eLBati) [inline mrp_repair_payment_term/static/description/index.html:1]: @hurrinico if you use README, you have to delete this file

**Review by eLBati — CHANGES_REQUESTED** [review]
Please use new API

**Comment by LoisRForgeFlow** [comment]
@hurrinico is this still relevant? are you going to finish the work? Thanks.

**Comment by LoisRForgeFlow** [comment]
Closing as this is stale. Please reopen if you think still is relevant.

**Pull request closed without merging**